config/routes.rb

```diff
@@ -11,6 +11,9 @@
   namespace :account do
     namespace :admin do
       root "user#index"
+      get "/users/", to: 'user#index'
```

Comment:
resources only: :index
https://guides.rubyonrails.org/routing.html#configuring-the-rails-router